### PR TITLE
Test for azure-pipelines

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by scopatz** [comment]
This actually passed, should I merge it?

File: .azure-pipelines/run_docker_build.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```
Comment (isuruf) [inline .azure-pipelines/run_docker_build.sh:1]: This file is duplicated?

Comment (mariusvniekerk) [reply .azure-pipelines/run_docker_build.sh:1]: yes.  It comes from the same template.  In the case where circle building is disabled we still want it to be generated.

Comment (mariusvniekerk) [reply .azure-pipelines/run_docker_build.sh:1]: Also this particular one is not exactly the same as the azure stuff has publishing artifacts disabled for now.

**Pull request closed without merging**